tests/unit/data_processing/test_data_processor.py

```diff
@@ -100,6 +111,18 @@ def test___init__(
         mock_default_transformers.assert_called_once()
         mock_regex_generator.assert_called_once()
 
+        expected_default_transformers = {
+            'numerical': 'FloatFormatter()',
+            'categorical': 'LabelEncoder(add_noise=True)',
+            'boolean': 'LabelEncoder(add_noise=True)',
+            'datetime': 'UnixTimestampEncoder()',
+            'id': 'RegexGenerator()',
+            'pii': 'AnonymizedFaker()',
+        }
+        assert 'text' not in data_processor._transformers_by_sdtype
+        assert 'id' in data_processor._transformers_by_sdtype
+        assert data_processor._transformers_by_sdtype == expected_default_transformers
```

Comment:
the last assert is probably all you need

Reply:
Yes, done in [9a01726](https://github.com/sdv-dev/SDV/pull/1506/commits/9a01726bfb8dc311c8525770a6f421cb38d0fb96)